recipes/gnuastro/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "gnuastro" %}
+{% set version = "0.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://ftp.gnu.org/gnu/{{name}}/{{name}}-{{version}}.tar.gz
+  sha256: 6d56dc161ecd80c64dbf09808f58114b52ad2ce648c71551b871f5d10d4c07c5
```

You could download `http://git.savannah.gnu.org/cgit/gnuastro.git/patch/?id=3fa6512537ea1905583cf2b7dff45d174260c4c3` and apply it as a patch instead of passing `--disable-buildprog`:
```suggestion
  sha256: 6d56dc161ecd80c64dbf09808f58114b52ad2ce648c71551b871f5d10d4c07c5
  patches:
    - remove-hardcoded-gcc.patch
```